zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -4796,6 +4837,83 @@ components:
         incidentKey:
           description: Incident key associated with this flow node instance.
           type: string
+    AdHocSubprocessActivitySearchQuery:
+      description: Ad-hoc subprocess activities search request.
+      type: object
+      required:
+        - filter
```

Comment:
For now, this is required, but we can loosen this later when implementing the enhanced query functionality in #28432.